deploy/osd-limited-support/not-in-limited-support/not-in-limited-support.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
```

Comment:
`deploy/osd-limited-support/config.yaml` and `deploy/osd-limited-support/in-limited-support/limited-support.yaml` are sufficient to deploy the configmap for a limited support cluster. We don't need a configmap for fully supported clusters, so this can be removed.

Reply:
Thanks! I wasn't sure. I'll remove them.